src/main/java/ch/securify/Main.java

```diff
@@ -121,14 +114,19 @@
             byte[] fileContent = Files.readAllBytes(new File(e.getKey().split(":")[0]).toPath());
 
             SolidityResult allPatternResults = CompilationHelpers.getMappingsFromStatusFile(livestatusfile, map, fileContent);
+
+            Gson gson = new GsonBuilder().setPrettyPrinting().create();
+            gson.toJson(allPatternResults, System.err);
```

Comment:
Why we output it: to indicate progress
Why to stderr: to try and keep a real json structure in stdout

Do you prefer something else?

Reply:
> How is this done today with this change?

Otherwise, I prefer a progress flag. Don't we need this functionality for the web interface, where having all debug in there would be an issue?

Reply:
How is this done today with this change?

Otherwise, I prefer a progress flag. Don't we need this functionality for the web interface, where having all debug in there would be an issue?

Reply:
The web interface uses `--livestatusfile` to save the results to a file and expose them, it does not rely on stdout so I don't think having all debug output there would be an issue.

So you'd rather have a new option `--progress` which, if enabled, will activate this change (but sent to stdout instead of stderr)?

Reply:
I'll implement what I just suggested, as discussed.